service/assessment/assessment.go

```diff
@@ -250,7 +243,7 @@ func (svc *Service) AssessEvidence(_ context.Context, req *assessment.AssessEvid
 	}
 
 	// Assess evidence
-	err = svc.handleEvidence(req.Evidence, resourceId)
+	_, err = svc.handleEvidence(req.Evidence, resourceId)
```

Comment:
Why are you retourning the assessment results in handleEvidence() if you not need them?

Reply:
We had one test that tested whether the results were created successfully. It kind of makes sense to return it, if somebody wants to have a look at it.

Reply:
You mean while debugging? That we can also do in the handle function, but ok.